nap/docker/ckan/ckan-plugins/ckanext-napote_theme/ckanext/napote_theme/validators.py

```diff
@@ -13,9 +13,18 @@
 def email_uniq_validator(value, context):
     model = context['model']
 
-    users = model.User.by_email(value)
+    username = context['user']
 
-    if users:
-        raise Invalid(_('Email already exists'))
+    try:
+        user_obj =  context['user_obj']
+    except KeyError:
+        user_obj = None
```

Comment:
No need to use try ... except here. You could do this: user_obj = context.get('user_obj') -> None if key does not exist